CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ IMPROVEMENTS:
 * `vcd_org_vdc` Add import capability - [#324]
 * `vcd_org_vdc` Add full read capability - [#324]
 * Upgrade Terraform SDK dependency to 0.12.8 [#320]
+* `resource/vcd_vapp_vm` have new field `computer_name` [#334]
 
 BUG FIXES:
```

Comment:
Haven't we decided to leave this old behaviour intact? I.e. set computer name _only_ if initscript is not empty (and use the new `computer_name` otherwise).

Reply:
I left. So won't add as bug fix comment. IMHO

Reply:
Oh my. Indeed.